Modules/Devices/test/Devices.Application.Tests/Tests/Devices/Commands/DeleteDevice/HandlerTests.cs

```diff
@@ -60,20 +60,17 @@ public void Deleting_a_device_not_owned_by_the_current_identity_is_not_possible(
     {
         // Arrange
         var identity1 = TestDataGenerator.CreateIdentity();
-        var onboardedDevice1 = CreateOnboardedDevice(identity1);
+        var onboardedDevice = CreateOnboardedDevice(identity1);
+        onboardedDevice.User.LoginOccurred();
```

Comment:
Why do you call `LoginOccurred` again? It is already called in `CreateOnboardedDevice`